zenlink-protocol/src/lib.rs

```diff
@@ -984,5 +991,19 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(18)]
+		#[pallet::weight(100_000_000)]
```

Comment:
Could do `T::DbWeight::get.writes(1)` instead

Reply:
just wanna keep it consistent with the rest. 100_000_000 is more than generous and this is a root operation,